### PR TITLE
add poll_frequency_seconds to snowflake queries (#28)

### PR DESCRIPTION
**Pull request by thesubneo**
## Summary
Current implementation cannot run ~10 queries concurrently. 
## Relevant Issue(s)
#28 

## Checklist
- [x] Summarized PR's changes in [CHANGELOG.md](https://github.com/PrefectHQ/prefect-snowflake/blob/main/CHANGELOG.md)

### CODE REVIEW

**Review by suresh-aqfer — APPROVED** [review]

**Comment by desertaxle** [comment]
Thanks for the contribution @thesubneo! Once the changelog update in #30 is included in this PR, this will be good to merge!

**Comment by thesubneo** [comment]
I have updated CHANGELOG.md here

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## Unreleased
 
 ### Added
+- `poll_frequency_seconds` to `snowflake_query` and `snowflake_multiquery`.
+Number of seconds to wait in between checks for run completion.
```
Comment (ahuang11) [inline CHANGELOG.md:12]:
```suggestion
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -8,6 +8,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## Unreleased
 
 ### Added
+- `poll_frequency_seconds` to `snowflake_query` and `snowflake_multiquery`.
```
Comment (ahuang11) [inline CHANGELOG.md:11]:
```suggestion
- `poll_frequency_seconds` to `snowflake_query` and `snowflake_multiquery` [#29](https://github.com/PrefectHQ/prefect-snowflake/pull/29).
```

**Review by ahuang11 — CHANGES_REQUESTED** [review]
Just minor suggestion

**Review by ahuang11 — APPROVED** [review]
Awesome!! Thanks so much for adding this

**Pull request merged**